applications/FluidDynamicsApplication/python_scripts/vms_fractional_step_solver.py

```diff
@@ -252,9 +251,9 @@ def Solve(self):
 # self.solver.ApplyFractionalVelocityFixity()
             (self.neighbour_search).Execute()
 # self.slip_conditions_initialized = False
-        if self.use_slip_conditions:
-            self.normal_util.CalculateOnSimplex(
-                self.model_part, self.domain_size, IS_STRUCTURE)
```

Comment:
I guess that you left this one for another PR. Am I wrong?

Reply:
I will continue to work on this PR some more, but I do not know if we want to change this one. Current Fluid scripts use the version that computes the normals on an entire (sub)-modelpart, so we may want to leave this one for other applications (incompressible, ulf) which still use the variable.

I could also add an overload to `CalculateOnSimplex` that replaces the checks for IS_STRUCTURE with checks for a flag.

Reply:
I would do something like that. In any case this solver is not compatible with the json processes so we could rely that the users set the IS_STRUCTURE variable if they use it.